processors/include/ClusterOnTrackAnaProcessor.h

```diff
@@ -5,7 +5,7 @@
 #include "HpsEvent.h"
 #include "Track.h"
 #include "TrackerHit.h"
-
+#include "Collections.h"
```

Comment:
Why are you adding this, we are trying to deprecate Collections.h. Try removing it and see if anything breaks without it.